### ISSUE TITLE
Setting indent makes stacktraces harder to read

### ISSUE THREAD

**Issue by jotomo** [pmwmedia/tinylog#24]
Without having _indent_ set in the _formatting_ specs, a stacktrace logs like this:
```
02:11:41 DEBUG    [ForkJoinPool-1-worker-9  ] Batch                    :75 : java.net.SocketTimeoutException: Read timed out
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at 
[...]	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
[...]
```
Setting the _formatting_ to something like _...{message|indent=2}_ yields this output:

```
02:00:07 DEBUG    [ForkJoinPool-1-worker-15 ] Batch                    :75 : java.net.SocketTimeoutException: Read timed out
  at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
  at 
[...]
  at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
  Caused by: java.net.SocketTimeoutException: Read timed out
  at java.net.SocketInputStream.socketRead0(Native Method)
[...]
```
Note how the _cause_ exception header is indented by two spaces in the second example, as are all other lines. This removes the visual clue that allows to quickly jump between exceptions when reading through a stracktrace. Is this by design (indent hard-aligns everything by amount specificed), cause I find the visual clue very helpful ...

**Comment by jotomo** [comment]
Intuitively, for printing stacktraces, I would have expected an indent of two to mean to add two spaces before each line, plus indents specific to the stacktrace, so that an exception header would be indented with two spaces and method/lines references with two or four additional spaces.

**Comment by pmwmedia** [comment]
I agree with you. This can be implemented exactly as you have described for tinylog 1.1.

**Comment by pmwmedia** [comment]
Fixed by commit 9a981e074aa175bb48a1bb54b4f75873c742d467
